bundles/org.openhab.core.addon.eclipse/pom.xml

```diff
@@ -25,6 +25,11 @@
       <artifactId>org.openhab.core.addon</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Why does the eclipse bundle need this dependency? There do not seem to be any changes to that bundle?